components/Global/styles.js

```diff
@@ -0,0 +1,104 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  :root {
+    --color-main: #334BC8;
+    --color-main-light: #2196F3;
+    --color-main-dark: #001AA3;
+    --color-neutral: #FAFAFA;
+    --color-neutral-dark: #7B7B7B;
+    --color-dark: #000;
+    --color-light: #fff;
+  }
+  html, body, div, span, applet, object, iframe,
```

Comment:
Imagino que esse seja o reset do Eric Meyer, que de fato é famoso e bom. Recentemente eu conheci esse: https://elad2412.github.io/the-new-css-reset/

É genial e mais leve xD

Reply:
Caracas, não conhecia esse... Realmente é melhor, vou adotar para os meus projetos. Obrigado professor!